aldryn_faq/models.py

```diff
@@ -70,17 +76,21 @@ def model_type_id(self):
         return ContentType.objects.get_for_model(self.__class__).id
 
     def get_absolute_url(self, language=None, slug=None):
-        if language is None:
-            language = get_current_language()
+        language = language or get_current_language()
 
-        if slug is None:
+        if not slug:
```

Comment:
This is precisely why I made the change. A falsey slug is an invalid slug, so, we'll create it from the object.